backend/src/features/hall-of-fame/hall-of-fame.service.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { HallOfFameService } from './hall-of-fame.service';
+import { TestDbService } from '../../../test/utils/test.db.service';
+import { PrismaService } from '../../prisma.service';
+import { ScoreEntryLevel } from '@prisma/client';
+import { ScoreEntryDto } from './model/score.entry.dto';
+import { TestDataUtil } from '../../../test/utils/test.data.util';
+
+const testDbService = new TestDbService();
+
+describe('HallOfFameService', () => {
+  let service: HallOfFameService;
```

Comment:
müsste das nicht auch mit `const` gehen? du schreibst den wert ja nur einmal im `beforeEach`

Reply:
mit const müsste ich es direkt deklarieren, was ich ja in beforEach dann mache.

Reply:
alles klar